### PR TITLE
adwaita-qt: 1.1.4 -> 1.3.0

### PR DESCRIPTION
**Pull request by dasj19**
###### Motivation for this change
Version bump. Fixed dependencies and the "substituteInPlace" path.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 123629` at e4dc552a run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>adwaita-qt</li>
    <li>tests.trivial</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: unused-argument

Unused argument: `pantheon`.
Near pkgs/data/themes/adwaita-qt/default.nix:9:3:
```
  |
9 | , pantheon
  |   ^
```
</li>
  </ul>
</details>

**Comment by jtojnar** [comment]
According to changelogs, it should now also support other platforms than Linux.

**Comment by jtojnar** [comment]
Additionally, xorg.libxcb is now required on Linux:

https://github.com/FedoraQt/adwaita-qt/blob/b348762c7d5f92fb8cbbcb781c94ed675bf916f8/CMakeLists.txt#L25

**Comment by dasj19** [comment]
@jtojnar Thanks for the review. I applied your suggestions. However I can not test the package on other platforms than linux (32 and 64 bit)

File: pkgs/data/themes/adwaita-qt/default.nix [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
+{ mkDerivation
+, stdenv
+, lib
+, fetchFromGitHub
+, nix-update-script
+, cmake
+, ninja
+, qtbase
+, qt5
+, xorg ? stdenv.isLinux
```
Comment (jtojnar) [inline pkgs/data/themes/adwaita-qt/default.nix:10]: This sets a default value to a boolean. Not necessary.

File: pkgs/data/themes/adwaita-qt/default.nix [hunk 2]
```diff
@@ -18,11 +28,13 @@ mkDerivation rec {
 
   buildInputs = [
     qtbase
+    qt5.qtx11extras
+    xorg.libxcb
```
Comment (jtojnar) [inline pkgs/data/themes/adwaita-qt/default.nix:32]: Instead, you would use a conditional here:

```suggestion
  ] ++ lib.optionals stdenv.isLinux [
    xorg.libxcb
```

Comment (dasj19) [reply pkgs/data/themes/adwaita-qt/default.nix:33]: Thanks, applied

**Review by jtojnar — APPROVED** [review]

**Pull request merged**